codegen/templates/tchannel_client.tmpl

```diff
@@ -266,7 +267,15 @@ type {{$clientName}} struct {
 		} else {
 			// We want hystrix ckt-breaker to count errors only for system issues
 			var clientErr error
+			scope := c.defaultDeps.Scope.Tagged(map[string]string{
+			"client" : "{{$svc.Name}}",
+			})
+		  start := time.Now()
 			err = hystrix.DoC(ctx, "{{$clientID}}", func(ctx context.Context) error {
+				t := time.Now()
+			  elapsed := t.Sub(start)
+			  size := scope.Timer("hystrix-timer")
```

Comment:
`size` doesn't suit here, probably timer. You may even just do `scope.Timer("hystrix-timer").Record(elapsed)` without requiring a new var.